jest.config.js

```diff
@@ -17,8 +12,12 @@ module.exports = {
             statements: 100,
         },
     },
+    transform: {
+        '^.*\.ts$': ['ts-jest', {
+            tsconfig: 'test/tsconfig.json'
+        }]
+    },
```

Comment:
These could likely be removed since the arrays I'm parsing above are typed as any[]. I could just as easily type the arguments as arg0: T[0] | T, argN: ...T but using these types are technically truer to what's going on.